deploy/mayastor-daemonset.yaml

```diff
@@ -125,6 +130,10 @@ spec:
         hostPath:
           path: /dev
           type: Directory
+      - name: host-root
+        hostPath:
+          path: /
+          type: Directory
```

Comment:
Yeah, I don't like it either. It is the price we pay to use iscsiadm without having distribution specific yaml files. 
From the running iscsiadm P.O.V it is more correct than other soultions, since we are able to reproduce the host environment more closely.

Reply:
agree. As discussed offline, it seems ok to me now 👍